cime_config/machines/config_compilers.xml

```diff
@@ -86,29 +86,32 @@ flags should be captured within MPAS CMake files.
 
 <compiler COMPILER="cray">
   <CFLAGS>
-    <append compile_threaded="FALSE"> -h noomp </append>
+    <append compile_threaded="TRUE"> -h omp </append>
```

Comment:
Just wondering if we should have a noomp build option to completely turn off OpenMP for non-threaded builds. We can still test with single and multiple threads.

Reply:
`-h noomp` is now default (it may have been the other way around previously): both with Cray Fortran v10.0.1 on Theta and v11.0.0 on Tulip.